api/timerange.go

```diff
@@ -53,8 +53,8 @@ func (tr Timerange) End() int64 {
 }
 
 func (tr Timerange) EndTime() time.Time {
-	seconds := tr.End() / 1000
-	milliseconds := tr.Start() % 1000
+	seconds := tr.end / 1000
+	milliseconds := tr.end % 1000
```

Comment:
This line was actually wrong before- it caused a test to suddenly start failing when I made a (seemingly benign) change.